traitcuration/traits/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+class Trait(models.Model):
+    name = models.CharField(max_length=200)
+    current_mapping = models.ForeignKey(
+        'Mapping', on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
```suggestion
    current_mapping = models.ForeignKey('Mapping', on_delete=models.SET_NULL, null=True, blank=True)
```

This fits on one line in less than 120 characters

Reply:
@joj0s As far as I see, this hasn't been resolved yet. Granted, this is tiny, but if we're following 120 character line limit, then we might as well be consistent :)